src/__init__.py

```diff
@@ -13206,6 +13243,9 @@ def xref(self):
 
     rect = property(bound, doc="page rectangle")
 
+    # any result of layout analysis is stored here
+    layout_information: typing.Optional[typing.List[tuple]] = None
```

Comment:
Do we actually need the typing hint  here? It makes this simple line of code look very complicated, and presumably this is internal implementation, not for users?

Reply:
Ok ... then let's simplify this ...